go/pkg/yolosvc/driver_github.go

```diff
@@ -103,6 +105,27 @@ func (svc *service) GitHubWorker(ctx context.Context, opts GithubWorkerOpts) err
 	}
 }
 
+func parseRepoConfigs(repos string) []githubRepoConfig {
+	if repos == "" {
+		return []githubRepoConfig{}
```

Comment:
```suggestion
		return nil
```

nil is better because it can be checked + it won't break the program if you just do `for repo := range repos`

Reply:
and more readable